Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the siteconfig-manager binary
-FROM registry.redhat.io/ubi9/go-toolset:1.22.9 AS builder
+FROM registry.ci.openshift.org/stolostron/builder:go1.23-linux AS builder
```

Comment:
IIRC you can't use `registry.ci.openshift.org` with konflux builds, but maybe that changed?